lexos/static/js/scripts_content_analysis.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Created by alvaro on 9/23/17.
+ */
```

Comment:
add document to your javascript.

you just type `/**<RET>` pycharm will generate the document template